python/mlcroissant/mlcroissant/_src/core/git.py

```diff
@@ -37,7 +37,9 @@ def download_git_lfs_file(file: Path):
     try:
         repo.execute(["git", "lfs", "pull", "--include", fullpath])
     except deps.git.exc.GitCommandError as ex:
-        raise RuntimeError("Problem when launching `git lfs`. "
-                           "Possible problems: Have you installed git lfs "
-                           f"locally? Is '{fullpath}' a valid `git lfs` "
-                           "repository?") from ex
+        raise RuntimeError(
+            "Problem when launching `git lfs`. "
+            "Possible problems: Have you installed git lfs "
+            f"locally? Is '{fullpath}' a valid `git lfs` "
```

Comment:
Wouldn't it be more readable if it would display `Possible problems:\n\t* Have you installed ...?\n\t* Is '{fullpath}' ...? `

Reply:
This PR only aims at reformating the code using the last version of black